src/libraries/System.Text.Json/ref/System.Text.Json.InboxOnly.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The rationale was documented in the API issue and was to avoid adding a `System.Linq.Expressions` package reference plus [dependency closures](https://www.nuget.org/packages/System.Linq.Expressions/4.3.0). We've purposely avoided SLE in the past for this reason, but "dynamic" support requires it.

The hard dependency on netstandard is no longer there -- messaging around "code sharing" has changed from using netstandard to using .NET 5.0.

The basic question is whether all new STJ functionality also needs to be in netstandard. Eventually that will no longer be possible as new functionality is added like utf8string.

Reply:
Are you sure a package reference would be needed?  I think all of S.L.E is part of netstandard2.0.  Package was from netstandard1.x era.  This does make S.T.J sit higher in the stack, or at least the part that uses dynamic, but that shouldn't really be any different for ns2.0 as it is for net5.0.  I think having the dynamic support work downlevel is important.

Reply:
After looking at this further, no package reference needs to be added for netstandard2.0. A new commit adds dynamic for all configurations.